FirebaseRemoteConfig.podspec

```diff
@@ -44,7 +43,6 @@ app update.
   s.dependency 'FirebaseInstallations', '~> 1.1'
   s.dependency 'GoogleUtilities/Environment', '~> 6.2'
   s.dependency 'GoogleUtilities/NSData+zlib', '~> 6.2'
-  s.dependency 'Protobuf', '~> 3.9', '>= 3.9.2'
```

Comment:
do we still need this for abt?

Reply:
ABT defines the dependency itself. We still need the Protobuf related define flag though for including the protobuf headers from ABT.